<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HBCISettingsUi</class>
 <widget class="QDialog" name="HBCISettingsUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AqHBCI Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="introPage">
      <attribute name="title">
       <string>Intro</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="Q3GroupBox" name="groupBox3">
         <property name="title">
          <string>Welcome</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="Q3TextBrowser" name="textBrowser1">
            <property name="text">
             <string>&lt;qt&gt;
&lt;h2&gt;&lt;font color=&quot;blue&quot;&gt;Welcome to AqHBCI's Setup Tool&lt;/font&gt;&lt;/h2&gt;

&lt;h3&gt;What is AqHBCI?&lt;/h3&gt;
&lt;p&gt;AqHBCI is a backend for the &lt;i&gt;AqBanking&lt;/i&gt; library. Together, these libraries provide support for online banking via &lt;i&gt;HBCI&lt;/i&gt;. HBCI is an online banking protocol (&lt;b&gt;H&lt;/b&gt;ome &lt;b&gt;B&lt;/b&gt;anking 
&lt;b&gt;C&lt;/b&gt;omputer &lt;b&gt;I&lt;/b&gt;nterface) widely used in Germany.
&lt;/p&gt;

&lt;h3&gt;What Can I Do With This Tool? &lt;/h3&gt;
&lt;p&gt;
With this tool, you can create or import HBCI users and accounts. On the tab &quot;Users&quot;, click &quot;New&quot; to create or import a new HBCI user.
&lt;/p&gt;
&lt;p&gt;
Many buttons have a built-in &lt;b&gt;help&lt;/b&gt; text. Press the &lt;i&gt;&quot;What's this?&quot;&lt;/i&gt; button below and then the button you need help with. If there is a help text for the selected button it will be shown.
&lt;/p&gt;

&lt;h4&gt;Importing Existing Users&lt;/h4&gt;
&lt;p&gt;
AqHBCI is able to import key files or chip card users created with the following programs:
&lt;ul&gt;
&lt;li&gt;GnuCash&lt;/li&gt;
&lt;li&gt;AqMoney&lt;/li&gt;
&lt;li&gt;AqMoney2&lt;/li&gt;
&lt;li&gt;GOpenHBCI&lt;/li&gt;
&lt;li&gt;KOpenHBCI&lt;/li&gt;
&lt;li&gt;LxBank&lt;/li&gt;
&lt;/ul&gt;

All these programs use one of the libraries OpenHBCI, OpenHBCI, or AqBanking for online banking support.
&lt;/p&gt;

&lt;p&gt;
You can still use these programs &lt;b&gt;and&lt;/b&gt; AqBanking-aware programs in parallel.
&lt;/p&gt;

&lt;h4&gt;Creating New Users&lt;/h4&gt;
&lt;p&gt;
You can create new users with the following HBCI security media:
&lt;ul&gt;
&lt;li&gt;&lt;b&gt;RDH&lt;/b&gt; (Self-generated key file)&lt;/li&gt;
&lt;li&gt;&lt;b&gt;DDV&lt;/b&gt; (Chip card)&lt;/li&gt;
&lt;li&gt;&lt;b&gt;PIN/TAN&lt;/b&gt;&lt;/li&gt;
&lt;/ul&gt;
&lt;/p&gt;
&lt;p&gt;
The security medium to use is determined by your credit institute.
&lt;/p&gt;
&lt;p&gt;
When using &lt;b&gt;RDH&lt;/b&gt; you should use the &lt;b&gt;OpenHBCI-plugin&lt;/b&gt; as key file if you want to use it with the programs mentioned above in parallel.
&lt;/p&gt;

&lt;/qt&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="userPage">
      <attribute name="title">
       <string>Users</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="Q3GroupBox" name="userBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Users</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QPushButton" name="newUserButton">
             <property name="toolTip">
              <string>Start the user wizard</string>
             </property>
             <property name="whatsThis">
              <string>&lt;qt&gt;
This button starts the user wizard which lets you &lt;i&gt;create&lt;/i&gt; or &lt;i&gt;import&lt;/i&gt; 
existing users.
&lt;/qt&gt;</string>
             </property>
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="delUserButton">
             <property name="toolTip">
              <string>Remove a user</string>
             </property>
             <property name="whatsThis">
              <string>&lt;qt&gt;
&lt;p&gt;
Removes the currently selected user.
&lt;/p&gt;
&lt;p&gt;
Handle this with care: If you remove a user which still is in use then the accounts 
accessed via this user become unavailable.
&lt;/p&gt;
&lt;p&gt;
However, the security media (key file, chip card) are not deleted, so you may later
import a deleted user again via the security medium.
&lt;/p&gt;
&lt;/qt&gt;</string>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="iniLetterButton">
             <property name="toolTip">
              <string>Only show the Ini-Letter of this user</string>
             </property>
             <property name="text">
              <string>Ini-Letter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Q3GroupBox" name="groupBox17">
             <property name="title">
              <string>Online Actions</string>
             </property>
             <layout class="QVBoxLayout">
              <item>
               <widget class="QPushButton" name="completeUserButton">
                <property name="toolTip">
                 <string>Complete a pending setup</string>
                </property>
                <property name="whatsThis">
                 <string>&lt;qt&gt;
Press this button to finish the setup of a select user marked as &lt;i&gt;pending&lt;/i&gt;.
&lt;/qt&gt;</string>
                </property>
                <property name="text">
                 <string>Complete</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="changeVersionButton">
                <property name="toolTip">
                 <string>Change the HBCI protocol version to be used for this user</string>
                </property>
                <property name="whatsThis">
                 <string>&lt;qt&gt;
&lt;p&gt;
Changes the HBCI protocol version to be used for this user.
&lt;/p&gt;
&lt;p&gt;
This will connect to the server to update the bank info (which with most 
banks depends on the HBCI protocol version used).
&lt;/p&gt;
&lt;/qt&gt;</string>
                </property>
                <property name="text">
                 <string>Change Version</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="updateBPDButton">
                <property name="toolTip">
                 <string>Update the bank information for this user</string>
                </property>
                <property name="whatsThis">
                 <string>&lt;qt&gt;
&lt;p&gt;
Updates the information we have about the bank used by the selected 
user.
&lt;/p&gt;
&lt;p&gt;
This will connect to the server.
&lt;/p&gt;
&lt;/qt&gt;</string>
                </property>
                <property name="text">
                 <string>Update Info</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="Q3GroupBox" name="groupBox18">
             <property name="title">
              <string>Expert Actions</string>
             </property>
             <layout class="QVBoxLayout">
              <item>
               <widget class="QPushButton" name="editUserButton">
                <property name="text">
                 <string>Edit</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="spacer43">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>16</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="accountPage">
      <attribute name="title">
       <string>Accounts</string>
      </attribute>
      <layout class="QHBoxLayout">
       <item>
        <widget class="Q3GroupBox" name="accountBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Accounts</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QPushButton" name="newAccountButton">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editAccountButton">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delAccountButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer44">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>What's This?</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>(c) 2005 M.Preuss</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Horizontal Spacing2_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>Q3TextEdit</extends>
   <header>Qt3Support/Q3TextBrowser</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>HBCISettingsUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
